### ISSUE TITLE
-DHWLOC_ENABLE=OFF causes compilation failure

### ISSUE THREAD

**Issue by sheepman0** [fireice-uk/xmr-stak#1231]
Tried on multiple systems (Ubuntu and CentOS) but it seems if I use the -DHWLOC_ENABLE=OFF option it fails to build with the following error:

```
[ 42%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/minethd.cpp.o
In file included from /home/sysadmin/xmr-stak/xmrstak/backend/cpu/minethd.cpp:43:0:
/home/sysadmin/xmr-stak/xmrstak/backend/cpu/autoAdjust.hpp: In member function ‘bool xmrstak::cpu::autoAdjust::printConfig()’:
/home/sysadmin/xmr-stak/xmrstak/backend/cpu/autoAdjust.hpp:41:23: error: ‘class jconf’ has no member named ‘IsCurrencyMonero’
   if(::jconf::inst()->IsCurrencyMonero())
                       ^
/home/sysadmin/xmr-stak/xmrstak/backend/cpu/autoAdjust.hpp:43:20: error: ‘MONERO_MEMORY’ was not declared in this scope
    hashMemSizeKB = MONERO_MEMORY / 1024u;
                    ^
/home/sysadmin/xmr-stak/xmrstak/backend/cpu/autoAdjust.hpp:48:20: error: ‘AEON_MEMORY’ was not declared in this scope
    hashMemSizeKB = AEON_MEMORY / 1024u;
                    ^
CMakeFiles/xmr-stak-backend.dir/build.make:158: recipe for target 'CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/minethd.cpp.o' failed
make[2]: *** [CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/minethd.cpp.o] Error 1
CMakeFiles/Makefile2:104: recipe for target 'CMakeFiles/xmr-stak-backend.dir/all' failed
make[1]: *** [CMakeFiles/xmr-stak-backend.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

```
Any advice?

**Comment by psychocrypt** [comment]
This is a bug and it is solved in #1219 will be merged soon to te dev branch

sheepman0 <notifications@github.com> schrieb am Di., 27. März 2018, 17:09:

**Comment by sheepman0** [comment]
Ok that worked, thanks for all your hard work!
